client/src/pages/Home.js

```diff
@@ -1,20 +1,66 @@
-import React from 'react';
-import {Link} from 'react-router-dom';
-import {Button, Typography} from 'antd';
-
-const {Title} = Typography;
+import React, { useCallback, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Typography } from 'antd';
 
+const { Title } = Typography;
 
 const Home = () => {
-    return (
-        <div>
-            <Title style={{alignItems: 'center'}}>Startseite</Title>
-            <Title level={3}>Das ist unsere Startseite - GEIL!</Title>
-            <Link to='/modeler'>
-                <Button type='primary'>Go to Modeler</Button>
-            </Link>
-        </div>
-    );
-}
+  const [message, setMessage] = useState(null);
+  /* const [isFetching, setIsFetching] = useState(false);
+  const [url, setUrl] = useState('/Test');
+
+  const fetchData = useCallback(() => {
+    fetch('/test')
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`status ${response.status}`);
+        }
+        return response.json();
+      })
+      .then((json) => {
+        setMessage(json.message);
+        setIsFetching(false);
+      })
+      .catch((e) => {
+        setMessage(`API call failed: ${e}`);
+        setIsFetching(false);
+      });
+  }, [url]);
+
+  useEffect(() => {
+    setIsFetching(true);
+    fetchData();
+  }, [fetchData]); */
+
+  async function postData(url = '', data = {}) {
```

Comment:
Is this used?

Reply:
I would also love a comment on what this function does or a more explaining function name